osu.Game.Tests/Visual/Online/TestSceneBeatmapSetOverlay.cs

```diff
@@ -403,8 +408,314 @@ public void TestLocalBeatmaps()
                             },
                         },
                     },
-                });
+                }, false);
+            });
+
+            downloadAssert(true);
+        }
+
+        [Test]
+        public void TestUnavailable()
+        {
+            AddStep(@"show parts-removed", () =>
+            {
+                overlay.ShowBeatmapSet(new BeatmapSetInfo
```

Comment:
Why is it necessary to include this huge thing? Why not use the existing one and just set the availability?

Reply:
or having a small function that creates a test beatmapset, will do that

Reply:
Would be great if you can minimise the other one or two cases a full beatmapset is included (the ones that already existed before this PR).